### PR TITLE
[iOS] Implements custom navigation bar

### PR DESCRIPTION
**Pull request by YGuerin**
GitHub Issue (If applicable): #

## PR Type
What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

The command bar has not the good on every device

## What is the new behavior?

The command bar height is set to the device one

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

https://nventive.visualstudio.com/Umbrella/_workitems/edit/134573

### CODE REVIEW

File: src/Uno.UI/Controls/CommandBarRenderer.iOS.cs [hunk 1]
```diff
@@ -111,7 +121,12 @@ protected override void Render()
 				: null;
 			Native.BackIndicatorImage = backButtonIcon;
 			Native.BackIndicatorTransitionMaskImage = backButtonIcon;
+
+
```
Comment (davidjohnoliver) [inline src/Uno.UI/Controls/CommandBarRenderer.iOS.cs]: extra space

File: src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBar.cs [hunk 2]
```diff
@@ -8,6 +8,8 @@
 using Windows.UI.Xaml.Markup;
 using Windows.UI.Xaml.Controls.Primitives;
 using Uno.Logging;
+using UIKit;
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBar.cs]: This won't compile on other platforms. You'll need to add conditionals. (FindFirstChild is in the Uno.UI namespace on Android, I think on other platforms it's a normal method on FrameworkElement.)

File: src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBar.cs [hunk 3]
```diff
@@ -36,6 +38,8 @@ public CommandBar()
 			CommandBarTemplateSettings = new CommandBarTemplateSettings(this);
 		}
 
+		internal NativeCommandBarPresenter _presenter { get; set; }
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBar.cs]: Presenter

File: src/Uno.UWP/UI/ViewManagement/ApplicationView.iOS.cs [hunk 4]
```diff
@@ -17,17 +17,16 @@ internal void SetVisibleBounds(UIKit.UIWindow keyWindow, Foundation.Rect windowB
...
 			// So we need to update the top inset depending of the status bar visibilty on other devices
-			if (UIDevice.CurrentDevice.UserInterfaceIdiom != UIUserInterfaceIdiom.Phone)
-			{
-				inset.Top = UIApplication.SharedApplication.StatusBarHidden
+			var statusBarHeight = UIApplication.SharedApplication.StatusBarHidden
 					? 0
 					: UIApplication.SharedApplication.StatusBarFrame.Size.Height;
-			}
+
+			inset.Top = (nfloat)Math.Max(inset.Top, statusBarHeight);
```
Comment (davidjohnoliver) [inline src/Uno.UWP/UI/ViewManagement/ApplicationView.iOS.cs]: different bug?

Comment (YGuerin) [reply src/Uno.UWP/UI/ViewManagement/ApplicationView.iOS.cs]: On iOS 11 and iPad, the status bar is not included in the safearea so we will set the visible bounds under the status bar for these devices. It's not the same one but related. I've noticed it when I was trying my stuff

File: src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBar.cs (hunk 2 above)
Comment (YGuerin) [reply src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBar.cs]: Actually the NativeCommandBarPresenter doesn't seems to exist on other platform. I just set it for Android and iOS. We can see that point tomorrow if needed

**Pull request merged**